backend/config/custom-environment-variables.json

```diff
@@ -99,7 +99,7 @@
     "clientSecret": "CROWD_GOOGLE_CLIENT_SECRET",
     "callbackUrl": "CROWD_GOOGLE_CALLBACK_URL"
   },
-  
```

Comment:
Maybe just remove this blank line so we don't have a random blank line in the middle =)